<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>239</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vyber let</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFare">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>239</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_FirstClass">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>239</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>First Class €150</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_BusinessClass">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>239</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Business Class €125</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_EconomyClass">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>239</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Economy Class €100</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_FirstClass</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>dispFare()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_BusinessClass</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>dispFare()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_EconomyClass</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>dispFare()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dispFare()</slot>
 </slots>
</ui>
